modules/topic_event_subscription/main.tf

```diff
@@ -0,0 +1,79 @@
+resource "azapi_resource" "event_subscription" {
```

Comment:
Please add `response_export_values = []` or be explicit abut the outputs you need in this attribute. This reduces the plan noise for module consumers.